src/SFA.DAS.AANHub.Application/Members/Commands/PostMemberStatus/PostMemberStatusCommandHandler.cs

```diff
@@ -0,0 +1,77 @@
+﻿using MediatR;
+using SFA.DAS.AANHub.Application.Common;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Domain.Common;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using System.Text.Json;
+
+namespace SFA.DAS.AANHub.Application.Members.Commands.PostMemberStatus;
+
+public class PostMemberStatusCommandHandler : IRequestHandler<PostMemberStatusCommand, ValidatedResponse<SuccessCommandResult>>
+{
+    private readonly IMembersWriteRepository _membersWriteRepository;
+    private readonly IAuditWriteRepository _auditWriteRepository;
+    private readonly IAanDataContext _aanDataContext;
+    private readonly INotificationsWriteRepository _notificationsWriteRepository;
+
+    public PostMemberStatusCommandHandler(IMembersWriteRepository membersWriteRepository, IAuditWriteRepository auditWriteRepository, IAanDataContext aanDataContext, INotificationsWriteRepository notificationsWriteRepository)
+    {
+        _membersWriteRepository = membersWriteRepository;
+        _auditWriteRepository = auditWriteRepository;
+        _aanDataContext = aanDataContext;
+        _notificationsWriteRepository = notificationsWriteRepository;
+    }
+
+    public async Task<ValidatedResponse<SuccessCommandResult>> Handle(PostMemberStatusCommand request,
+        CancellationToken cancellationToken)
+    {
+        var member = await _membersWriteRepository.Get(request.MemberId);
+
+        var audit = new Audit()
+        {
+            Action = "Post",
```

Comment:
Also to keep ourselves consistent, Action should come off an enum or constants.

Reply:
I'll create an enum and apply in all audit call actions